sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/FieldValueGetter.java

```diff
@@ -33,5 +33,9 @@
   @Nullable
   ValueT get(ObjectT object);
 
+  default @Nullable Object getRaw(ObjectT object) {
```

Comment:
nit: maybe `getUntyped` is a more appropriate name here?

Also could you clarify why we need this?

Reply:
Thanks so much for having a look @TheNeuralBit 🙏 

`getRaw()` was based on a [conversation](https://lists.apache.org/thread/gjq453fm32s76zlvjs4kb5g3rgxnh7gs) with @reuvenlax.

> getValues() is maybe poorly named - might be better called getRawValues. What you're  looking for is  probably the getBaseValues() method.
> getValues is mostly used in code that knows exactly what it's doing for  optimization purposes. It goes along with the attachValues method, which is similarly tricky to use. It's there to enable  0-copy  code, but not necessarily intended for general consumption. 

`RowWithGetters.getValues()`  returns the "raw" unmodified result of the getters:
```java
public List<Object> getValues() {
  return getters.stream().map(g -> g.get(getterTarget)).collect(Collectors.toList());
}
```

As I am pushing down the transformation of the getter result into the getter itself, I needed a way to bypass that in order to maintain the current semantics of `getValues()`. Let me know if the name makes sense given that context.

Reply:
Yeah that makes sense, thanks. Could you add some of this context in a comment there?

Reply:
@TheNeuralBit I've opened a new PR to add the missing comment, sorry for the delay.
https://github.com/apache/beam/pull/21982